### PR TITLE
Fixing NFS Issues

### PR DESCRIPTION
**Pull request by AmarnatReddy**
# Description
Fixing NFS Issues

Failed Logs:
http://magna002.ceph.redhat.com/cephci-jenkins/test-runs/18.2.0-127/Regression/cephfs/34/tier-2_cephfs_test-nfs/
It has 3 failures
1.[CEPH-83573992](https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83573992) -- Product bug --json-pretty is been not supported 
2. [CEPH-83574013](https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83574013) 
There is some change in behaviour 
earlier we used to get user_id in format as user_id = f"nfs.{cluster_id}.{export_id}" --> this has been changed to user_id = f"nfs.{cluster_id}.{fs_name}"

3. [CEPH-83575082](https://polarion.engineering.redhat.com/polarion/#/project/CEPH/workitem?id=CEPH-83575082)
This is an automation Bug where we are getting export_ids the same values because of this export creations were failing.
Changed the code to get 2 unique values 

Passed Log : 
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-RQTK6W/

Passed Log : http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-VZMJX7

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

**Review by julpark-rh — DISMISSED** [review]
overall LGTM

File: tests/cephfs/cephfs_nfs/nfs_info_cluster_id_and_ls_export_detailed.py [hunk 1]
```diff
@@ -54,10 +54,10 @@ def run(ceph_cluster, **kw):
         )
         cluster_id = f"test-nfs_{rand}"
         path = "/"
-        export_id = "1"
+        # export_id = "1"
```
[the remark this answers is not in the document]
Comment (sumabai) [reply tests/cephfs/cephfs_nfs/nfs_info_cluster_id_and_ls_export_detailed.py:57]: +1

**Review by sumabai — DISMISSED** [review]

**Review by hkadam134 — APPROVED** [review]
lgtm

**Review by sumabai — APPROVED** [review]

**Review by neha-gangadhar — APPROVED** [review]
Merging based on 2 approvals

**Pull request merged**